.circleci/config.yml

```diff
@@ -1,17 +1,22 @@
 version: 2.1
+
+orbs:
+  codacy: codacy/base@1.0.0
+
 jobs:
   build: # runs not using Workflows must have a `build` job as entry point
     docker: # run the steps with Docker
       # CircleCI Go images available at: https://hub.docker.com/r/circleci/golang/
       - image: circleci/golang:1.12 #
+    
     # directory where steps are run. Path must conform to the Go Workspace requirements
     working_directory: /go/src/github.com/codacy/helm-ssm
-
+    
```

Comment:
I think you forgot some leading space here

Reply:
Sorry, I merged before reading this comment.